kie-wb-webapp-common/src/main/webapp/login.jsp

```diff
@@ -24,7 +26,7 @@
   <link rel="stylesheet" href="styles/forms.css">
   <link rel="stylesheet" href="styles/login-screen.css">
   <link rel="shortcut icon" href="favicon.ico" />
-  <title><i18n:message key="LoginTitle">KIE Drools Workbench</i18n:message></title>
+  <title><i18n:message key="LoginTitle"></i18n:message></title>
```

Comment:
Does the login page still have a title, or have you removed it to simplify sharing the new module?